src/python/ServerUtilities.py

```diff
@@ -351,6 +353,9 @@ def isFailurePermanent(reason, gridJob=False):
         refuseToSubmit = ""
     for exitCode in STAGEOUT_ERRORS:
         for error in STAGEOUT_ERRORS[exitCode]:
+            if PY3:
```

Comment:
this is a perfect example of code which is used only in TW and could be "plain python3" w/o need of the "if PY3" switch. But mostly, why should we ever have bytes here if not because of a bug ? Are we going to need to wrap all calls to `re` with calls to decode ? Makes no sense IMHO.

Reply:
> Are we going to need to wrap all calls to re with calls to decode 
nono, you are correct. i am just slowly wandering through the code looking for the root cause. This PR is still a work in progress!

Reply:
I believe the cause of this was identified: https://github.com/dmwm/CRABServer/pull/6921#issuecomment-997047751

Reply:
yes, indeed that solved the problem, i had naive problems in the container (`./start.sh` does not seem to run `./stop.sh`, as in the `crabserver`)